modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -13,102 +13,25 @@
 
 """The module defines base interface for a partition of a Modin DataFrame."""
 
-from abc import ABC
 from copy import copy
 
 import pandas
 from pandas.api.types import is_scalar
 
 from modin.pandas.indexing import compute_sliced_len
 from modin.core.storage_formats.pandas.utils import length_fn_pandas, width_fn_pandas
+from modin.core.dataframe.base.partitioning.partition import BaseDataframePartition
 
 
-class PandasDataframePartition(ABC):  # pragma: no cover
+class PandasDataframePartition(
+    BaseDataframePartition
+):  # pragma: no cover; # noqa: PR01
```

Comment:
`PR01` is a mute of the doc checks of  `__init__` method, which is inherited from the base class.